Mixpanel/MPNetworkPrivate.h

```diff
@@ -34,5 +34,6 @@
                            withQueryItems:(NSArray <NSURLQueryItem *> *)queryItems
                                   andBody:(NSString *)body;
 
++ (NSURLSession *)sharedSession;
```

Comment:
very minor: would rather call this `sharedURLSession` or something so it can't be confused for some sort of `MPNetwork` session